spec/support/service_template_helper.rb

```diff
@@ -131,7 +131,6 @@ def service_template_stubs
       end
 
       def user_helper
-        allow_any_instance_of(User).to receive(:role).and_return("admin")
```

Comment:
sorry to do this but...

can you remove `user_helper` and put the `@user` call into the one place it is used [[service_template_filter_spec.rb]](https://github.com/kbrock/manageiq/blob/delegation_type/spec/models/service_template_filter_spec.rb#L6)